src/media/js/previews.js

```diff
@@ -28,17 +28,18 @@ define('previews',
 
     var desktopMarginLeft = 0;  // Keep track of left offset.
 
-    function setActiveBar($bars, position) {
-        $bars.filter('.current').removeClass('current');
-        $bars.eq(position).addClass('current');
-    }
-
-    function initUpdatePosition($bars, slider) {
+    function updatePreviewBarPos($bars, slider) {
         // Listener to update the colored bar position.
+        function setActiveBar($bars, position) {
+            if (slider._maxPoint > 2) {
+                // Only show bars if more than two screenshots.
+                $bars.filter('.current').removeClass('current');
+                $bars.eq(position).addClass('current');
```

Comment:
I don't think there's anything else that needs to be done with Flipsnap for this PR. good to review thanks